models/fixtures/training/bar.json

```diff
@@ -0,0 +1,93 @@
+{
+  "training": [
+    {
+      "role": "Bar",
+      "pass_auto": true,
+      "pass_mark": 100,
+      "url": "",
+      "questions": [
+        {
+          "question": "A proof of age document is reliable if it has",
+          "answers": [
+            {
+              "answer": "Local authority logo",
+              "correct": false
+            },
+            {
+              "answer": "PASS hologram",
+              "correct": true
+            },
+            {
+              "answer": "Police authority stamp",
+              "correct": false
+            }
+          ]
+        },
+        {
+          "question": "What is the on-the-spot fine for selling alcohol to someone who is under 18?",
+          "answers": [
+            {
+              "answer": "£60",
+              "correct": false
+            },
+            {
+              "answer": "£90",
+              "correct": true
+            },
+            {
+              "answer": "£120",
+              "correct": false
+            }
+          ]
+        },
+        {
+          "question": "I think someone may be under 18, but the bar manager has told me to serve them. Should I?",
+          "answers": [
+            {
+              "answer": "Yes",
+              "correct": false
+            },
+            {
+              "answer": "No",
+              "correct": true
+            }
+          ]
+        },
+        {
+          "question": "Cigarettes must not be sold to persons aged under?",
+          "answers": [
+            {
+              "answer": "18",
+              "correct": true
+            },
+            {
+              "answer": "21",
+              "correct": false
+            },
+            {
+              "answer": "25",
+              "correct": false
+            }
+          ]
+        },
+        {
+          "question": "Which of these is considered a common sign of drunkenness?",
```

Comment:
It's good that we require 100% to pass the test, but I worry that this question might be a little under-leveled.  It's useful as a pre-screen but I don't know if it'd stand up to strong scrutiny of the training.

Reply:
There are more questions in the full version of the test. I assume this file is a cut-down version for, er, testing.

Reply:
This isn't the training material, this is purely the test to confirm someone has read and understood the training material.

I might be having a lack of memory on the details at this point but I'm not sure if there were more than 3 questions last time?

Reply:
Ah, no, there's 5 here. I'n pretty sure that's what we had last time.

Reply:
I stand corrected, there were 12 answers in the original source:

https://github.com/emfcamp/Website/blob/master/apps/volunteer/bar_training.py#L79

So yes this is a partial list of both questions and answers. I'll add a comment to the ticket to make that clear.

Reply:
Thanks @sde1000 @idnorton!